### PR TITLE
Refactor NODE_ENV assertion function into test_helper

### PR DESCRIPTION
**Pull request by supasate**

### CODE REVIEW

File: test/app.test.js [hunk 1]
```diff
@@ -1,9 +1,13 @@
 'use strict';
 
+const assertTestEnv = require('./test_helper').assertTestEnv;
```
Comment (parnurzeal) [inline test/app.test.js]: Put below assert? (alphabetically)

File: test/services/photo/index.test.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
 'use strict';
 
+const assertTestEnv = require('../../test_helper').assertTestEnv;
```
Comment (parnurzeal) [inline test/services/photo/index.test.js]: Put below assert

File: test/services/pin/hooks/process.test.js [hunk 3]
```diff
@@ -1,8 +1,13 @@
 'use strict';
 
+const assertTestEnv = require('../../../test_helper').assertTestEnv;
+
 const assert = require('assert');
```
Comment (parnurzeal) [inline test/services/pin/hooks/process.test.js]: Put below assert

File: test/app.test.js (hunk 1 above)
Comment (parnurzeal) [inline test/app.test.js:3]: newline below this line.

File: test/services/pin/index.test.js [hunk 4]
```diff
@@ -1,7 +1,8 @@
 const app = require('../../../src/app');
 const casual = require('casual');
 const expect = require('../../test_helper').expect;
-const loadFixture = require('../../test_helper').loadFixture;
+const loadFixture = require('../../test_helper').loadFixture
+const assertTestEnv = require('../../test_helper').assertTestEnv;;
```
Comment (parnurzeal) [inline test/services/pin/index.test.js:5]: reorder alphabetically?

File: test/services/user/index.test.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 const app = require('../../../src/app');
 const casual = require('casual');
 const expect = require('../../test_helper').expect;
-const loadFixture = require('../../test_helper').loadFixture;
+const loadFixture = require('../../test_helper').loadFixture
+const assertTestEnv = require('../../test_helper').assertTestEnv;;
```
Comment (parnurzeal) [inline test/services/user/index.test.js:5]: reorder alphabetically?

**Comment by supasate** [comment]
@parnurzeal I'm thinking about grouping by purposes, then, ordering alphabetically within the group. Will refactor all files in another PR. What do you think?

**Comment by parnurzeal** [comment]
Merged.

**Pull request merged**